protocol/communities/manager.go

```diff
@@ -1966,6 +1937,25 @@ func (m *Manager) AcceptRequestToJoin(dbRequest *RequestToJoin) (*Community, err
 		return nil, err
 	}
 
+	if community.IsControlNode() {
+		// share accepted request to join with other privileged members
```

Comment:
I don't the comment is useful, since the method is named exactly the same :laughing: